app/views/posts/index.html.erb

```diff
@@ -1,2 +1,20 @@
-
-<h1>List of all posts by a user</h1>
\ No newline at end of file
+<div class="posts">
+ <%= render "shared/details", user: @user %>
+  <div>
+   <% @posts.each_with_index do |post| %>
+     <%= link_to user_post_path(user_id:@user.id, id: post.id) do%>
+       <%= render "shared/posts_details", post: post %>
+        <div class="comments">
+        <% if post.comments.empty? %>
+          <p>Empty</p>
+        <% else %>
+            <% post.comments.each do |comment| %>
+             <p class="comment-body"><%= comment.author.name %>: <%= comment.text %></p>
```

Comment:
- [ ]  Let's use the `most_recent_comments` callback function in our post model to render the 5 most recent comments for a post.